tests/basetest.rb

```diff
@@ -137,7 +137,23 @@ def teardown
     @device = nil
   end
 
+  # Execute the specified config commands and warn if the
+  # output matches the default "warning" regex.
   def config(*args)
+    config_and_warn_on_match(/invalid|^%/i, *args)
```

Comment:
Agreed, but since I don't have a list of all possible error message formats, and I am currently only logging warnings (vs errors), I thought it would be better to have a few false positives than false negatives, until we get more of a chance to go over the warnings.  I'm fine changing it, though, it others agree.